inc/class-wp-forms-api.php

```diff
@@ -925,6 +927,29 @@ static function process_element( $element, &$values, &$input ) {
 	}
 
 	/**
+	 * Get WP editor templates for dynamically-inserted editors in #multiple fields
+	 *
+	 * @action wp_ajax_wp_form_get_mce_templates
+	 */
+	function ajax_get_mce_templates() {
+		$results = array();
+		$bodies = filter_input( INPUT_POST, 'content', FILTER_DEFAULT, FILTER_REQUIRE_ARRAY );
+
+		foreach ( $bodies as $id => $body ) {
+			ob_start();
+			$lines = preg_split( '/$/m', $body );
+
+			wp_editor( $body, $id, array(
+				'textarea_name' => $id,
+				'textarea_rows' => count( $lines ),
+			)  );
+			$results[$id] = ob_get_clean();
```

Comment:
I have no idea about the performance implications but do you think it would be better to call `ob_start()` once outside of the loop and then `ob_get_contents()` and `ob_clean()` in the loop?